test/parallel/test-repl-persistent-history.js

```diff
@@ -111,6 +111,12 @@ const tests = [
     test: [UP],
     expected: [prompt, replFailedRead, prompt, replDisabled, prompt]
   },
+  { // Tests multiline history
+    env: {},
+    test: ['{', '}', UP, CLEAR],
+    expected: [prompt, '{', '... ', '}', '{}\n', prompt, `${prompt}{}`, prompt], // eslint-disable-line max-len
```

Comment:
nit: this really shouldn't need the exclusion... just go ahead and wrap.

Reply:
@jasnell Sure, will take care and push the change. Thanks!